backend/typescript/types.ts

```diff
@@ -169,29 +169,37 @@ export type CreateCampDTO = Omit<
   | "dropoffProductId"
   | "pickupPriceId"
   | "pickupProductId"
+  | "earlyDropoff"
+  | "latePickup"
+  | "pickupFee"
+  | "dropoffFee"
 > & {
   filePath?: string;
   fileContentType?: string;
   formQuestions: CreateFormQuestionDTO[];
   campSessions: CreateCampSessionsDTO;
+  earlyDropoff?: string;
+  latePickup?: string;
+  pickupFee?: number;
+  dropoffFee?: number;
 };
 
-export type UpdateCampDTO = Omit<
-  CampDTO,
-  | "id"
-  | "formQuestions"
-  | "campSessions"
-  | "campProductId"
-  | "dropoffPriceId"
-  | "dropoffProductId"
-  | "pickupPriceId"
-  | "pickupProductId"
-> & {
-  filePath?: string;
-  fileContentType?: string;
-  campSessions: CreateCampSessionsDTO;
-  formQuestions: CreateFormQuestionDTO[];
-};
+// export type UpdateCampDTO = Omit<
```

Comment:
what are we keeping this for? maybe add a comment on top for context

Reply:
removed